### PR TITLE
update plugin-store & manager responses

### PR DESCRIPTION
**Pull request by Pierregvx**

### CODE REVIEW

**Review by pivilartisant — APPROVED** [review]

File: api/swagger/server/restapi/resource/swagger.yml [hunk 1]
```diff
@@ -1050,12 +1055,18 @@ definitions:
...
+        type: string
       description:
         type: string
       version:
         type: string
       url:
         type: string
+      logo:
+        type: string
+      massastationVersion:
```
Comment (peterjah) [inline api/swagger/server/restapi/resource/swagger.yml:1068]: its weird to find massaStationVersion here.
it is not a property of a plugin.
Or maybe its the massaStation minimum version required ? in that case its poorly named

**Review by peterjah — CHANGES_REQUESTED** [review]
to follow

File: api/swagger/server/restapi/resource/swagger.yml (hunk 1 above)
Comment (Pierregvx) [reply api/swagger/server/restapi/resource/swagger.yml:1068]: it is part of th manifest https://github.com/massalabs/thyra-plugin-store/blob/04a13e300f78a704e23308413ed8a6315ce68e4e/plugins.json#L5
but not really used in the front end, I can remove it

Comment (peterjah) [reply api/swagger/server/restapi/resource/swagger.yml:1068]: ok! so its the minimum required version. can you just rename this ?
by instance `massastationMinVersion`

**Review by peterjah — APPROVED** [review]

**Pull request merged**